QUARKUS-5664.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-5664 TLS - Enable Policy Configuration for Expired or Not Yet Valid Certificates
+
+## Scope of testing
+Verify, that the user can configure actions for expired or invalid certificates in TLS registry 
```

Comment:
Here and below you use term `invalid` but I think as far as validity goes, only "not yet valid" invalidity is allowed. I think you need to be more specific, certificates can be invalid for other reasons.